docs/developers/environmental_considerations_and_pseudo_tags.md

```diff
@@ -47,9 +47,9 @@ OSM data recognized as forest:
 - landuse: forest, allotments, flowerbed, orchard, vineyard, recreation_ground, village_green
 - leisure: garden, park, nature_reserve
   
-No forest buffers are used.
+Forest has 77 m buffers.
```

Comment:
The are no buffers around the `osm_poly_no_buf` polygons in the `estimated_forest_class` calculation:
https://github.com/abrensch/brouter/blob/76265e7713ad203ee6679a810121d2bd22f78074/misc/scripts/mapcreation/brouter.sql#L585